### DISCUSSION TITLE
The repository tanuj1811 has been retired and cannot be reused

### DISCUSSION THREAD

**Discussion by tanuj1811** [community/community#23055] [New to GitHub]
<p><strong>Github Account Name</strong> : tanuj1811 <a href="https://github.com/tanuj1811**" rel="noopener nofollow ugc">https://github.com/tanuj1811**</a></p>
<p>Actually, I recently changed my username from tanuj1811 to tanujdadheech, and I also changed my special repo name on GitHub.</p>
<p>After some days I started found trouble as all my because everywhere I used my previous username So I changed again to my previous one, but there is an error on GitHub Repository name. So I accidentally delete my special repository.<br>
Now when I try to make new repo there error message pop up<br>
‘The repository tanuj1811 has been retired and cannot be reused.’</p>
<p>Can you please fix this asap.</p>
<p>Regards,</p>
<p>Tanuj Sharma</p>

**Comment by jsoref** [comment, marked as answer]
<p>File a ticket:</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://support.github.com/">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/2119212/181074601-0f602845-a349-41a0-9289-db3dde02cafa.png" class="site-icon" width="32" height="32">

      <a href="https://support.github.com/" target="_blank" rel="noopener">support.github.com</a>
  </header>

  <article class="onebox-body">
    

<h3><a href="https://support.github.com/" target="_blank" rel="noopener">GitHub Support</a></h3>

  <p>GitHub Support is here to help. Learn about GitHub products, browse our helpful resources, and contact support with your questions.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

**Comment by jfullstackdev** [comment]
<p>yup, it’s really not recommended to change username frequently, it’s your handle, so you can’t simply change it without side effects</p>

**Comment by queenofcorgis** [comment]
<p>Hi <a class="mention" href="/tanuj1811">@tanuj1811</a> appreciate you checking in, I’ve checked in and your ticket is in the proper place so rest assured it will be answered as soon as our team can get to it. Unfortunately, the Community Forum is unable to escalate or assist with tickets. Really appreciate your patience!</p>

**Comment by vishal-go** [comment]
<p>Now, same problem arise with me i have changed my github username from vishals39 to sharma39vishal . And now i am changing my special repository name to sharma39vishal it was showing "This repository sharma39vishal has been retired and cannot be reused ".so please help me to create special repository again.</p>

**Comment by jfullstackdev** [comment]
<p>seems that’s already fine,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/5/c518502c802f6b83032a97b92fbfc60c4b438c5f.png" data-download-href="/uploads/short-url/s7A9rUYvGpVVbwW2eMomCCw8lmT.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181074605-14d09289-4824-44d9-9543-1f3cf3641166.png" alt="image" data-base62-sha1="s7A9rUYvGpVVbwW2eMomCCw8lmT" width="690" height="373" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/5/c518502c802f6b83032a97b92fbfc60c4b438c5f_2_690x373.png, https://aws1.discourse-cdn.com/github/optimized/3X/c/5/c518502c802f6b83032a97b92fbfc60c4b438c5f_2_1035x559.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/c/5/c518502c802f6b83032a97b92fbfc60c4b438c5f_2_1380x746.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/5/c518502c802f6b83032a97b92fbfc60c4b438c5f_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1920×1040 107 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>it’s already displayed at your profile</p>

**Comment by vishal-go** [comment]
<p>Now it was solved by GitHub team thanks</p>

**Comment by yanfang-li** [comment]
<p><a class="mention" href="/liligalante">@liligalante</a> <a class="mention" href="/jdevstatic">@jdevstatic</a> I had the same issue and contacted github support 10 days ago, but I never got any response. My tickets’ IDs: 1705763 and 1699754. Could you help me to check if my tickets are in the proper place and when i can get a reply. Thank you in advance!</p>

**Comment by queenofcorgis** [comment]
<p>Hi there - I checked and we received your tickets, Support times may vary as they depend on our team’s resources and the number of tickets ahead of yours. Rest assured, they are in the proper place - thank you for your patience!</p>

**Comment by TechAndSam** [comment]
I have got the same issue. My ticket number is #1755383. Please help look into this ASAP.

**Reply by queenofcorgis** [reply]
Hi there - while I can confirm that we received your ticket, as a reminder, the Community team is unable to expedite tickets, but we appreciate your patience.

**Comment by nyaapass** [comment]
Same problem here. My ticket number is `#1722458`.

I understand it takes time to solve tickets, but I've been waiting for over **50 days**.

**Reply by queenofcorgis** [reply]
I appreciate your patience and understanding. I checked in on your ticket and can confirm it is in the right place, but the Community team is unable to interact with it beyond that. Hang tight!

**Comment by sxhil296** [comment]
I am having the same issue, can someone help me out in this?

**Reply by jsoref** [reply]
https://github.com/community/community/discussions/23055#discussioncomment-3238866
